Makefile

```diff
@@ -1,83 +1,190 @@
+    	fi
+
+## use this to test a new run on a totally new fresh environment after delete also aws resources
+test-complete: test-env-clean test-env-down remove-tf-state test-env-up test-run
+
 test-logs:
-	@docker logs -f acceptance-test${IMGNAME}
+	@if [ "${USE}" = "IMGNAME" ]; then \
+		docker logs -f acceptance-test-${IMGNAME}; \
+	elif [ "${USE}" = "TESTSTATE" ]; then \
+		docker logs -f acceptance-test-${TESTSTATE}; \
+	fi;
+
+image-stats:
+	@./scripts/docker_stats.sh $$IMGNAME 2>> /tmp/image-${IMGNAME}_stats_output.log &
 
 .PHONY: test-env-down
 test-env-down:
-	@echo "Removing containers and images"
-	@docker stop $$(docker ps -a -q --filter="name=acceptance-test*")
-	@docker rm $$(docker ps -a -q --filter="name=acceptance-test*") ; \
-	 docker rmi --force $$(docker images -q --filter="reference=acceptance-test*")
+	@echo "Removing containers"
+	@docker ps -a -q --filter="name=acceptance-test*" | xargs -r docker rm -f 2>/tmp/container_${IMGNAME}.log || true
+	@echo "Removing acceptance-test images"
+	@docker images -q --filter="reference=acceptance-test*" | xargs -r docker rmi -f  2>/tmp/container_${IMGNAME}.log  || true
+	@echo "Removing dangling images"
+	@docker images -q -f "dangling=true" | xargs -r docker rmi -f  2>/tmp/container_${IMGNAME}.log || true
+	@echo "Removing state images"
+	@docker images -q --filter="reference=teststate:latest" | xargs -r docker rmi -f  2>/tmp/container_${IMGNAME}.log  || true
 
 .PHONY: test-env-clean
 test-env-clean:
 	@./scripts/delete_resources.sh
 
-.PHONY: test-complete
-test-complete: test-env-clean test-env-down remove-tf-state test-env-up test-run
 
+#========================= Run acceptance tests locally =========================#
 .PHONY: remove-tf-state
 remove-tf-state:
-	@rm -rf ./modules/${PRODUCT}/.terraform
-	@rm -rf ./modules/${PRODUCT}/.terraform.lock.hcl ./modules/${PRODUCT}/terraform.tfstate ./modules/${PRODUCT}/terraform.tfstate.backup
-
+	@rm -rf ./modules/${ENV_PRODUCT}/.terraform
+	@rm -rf ./modules/${ENV_PRODUCT}/.terraform.lock.hcl ./modules/${ENV_PRODUCT}/terraform.tfstate ./modules/${ENV_PRODUCT}/terraform.tfstate.backup
 
-#========================= Run acceptance tests locally =========================#
 
 .PHONY: test-create
 test-create:
-	@go test -timeout=45m -v ./entrypoint/createcluster/...
+	@go test -timeout=45m -v -count=1 ./entrypoint/createcluster/...
+
 
 .PHONY: test-upgrade-suc
 test-upgrade-suc:
-	@go test -timeout=45m -v -tags=upgradesuc  ./entrypoint/upgradecluster/... -upgradeVersion ${UPGRADEVERSION}
+	@go test -timeout=45m -v -tags=upgradesuc -count=1 ./entrypoint/upgradecluster/... -sucUpgradeVersion ${SUCUPGRADEVERSION}
+
 
 .PHONY: test-upgrade-manual
 test-upgrade-manual:
-	@go test -timeout=45m -v -tags=upgrademanual ./entrypoint/upgradecluster/... -installVersionOrCommit ${INSTALLTYPE}
+	@go test -timeout=45m -v -tags=upgrademanual -count=1 ./entrypoint/upgradecluster/... -installVersionOrCommit ${INSTALLVERSIONORCOMMIT} -channel ${CHANNEL}
```

Comment:
INSTALL_VERSION_OR_COMMIT

Reply:
prefer that too

done

please do a overhaul with me on those vars, bc its easy to mess up and let some pass

Reply:
@VestigeJ